package.json

```
@@ -11,7 +11,8 @@
     "prepublishOnly": "npm run build",
     "bundle": "cd tools/bundler && npm i && npm run bundle",
     "startNewVersion": "newVersion=$npm_config_new_version node scripts/add-new-version.js",
-    "lint": "echo 'No linter integrated yet'",
+    "lint": "eslint --max-warnings 0 --config .eslintrs .",
```

```suggestion
    "lint": "eslint --max-warnings 0 --config .eslintrc .",
```
I think this is why the workflow failed earlier, @Min2who please fix this

---

Resolved with da9d4ce